Makefile.config.in

```diff
@@ -260,11 +260,12 @@ endif # ifeq "$(TOOLCHAIN)" "msvc"
 # User code should absolutely not rely on their presence because
 # in the future their definition may be moved to a more private part of
 # the compiler's build system
+DIFF = diff
 ifeq "$(UNIX_OR_WIN32)" "win32"
   OTOPDIR=$(WINTOPDIR)
   CTOPDIR=$(WINTOPDIR)
   CYGPATH=cygpath -m
-  DIFF=/usr/bin/diff -q --strip-trailing-cr
```

Comment:
These were fully qualified in ce969ca16a4f2583fc27e7a5107841756e527618 apparently to fix AppVeyor. I expect this may have been something weird with another set of Cygwin utilities in `PATH` on AppVeyor.